index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="style.css" rel="stylesheet">
+   
+    <title>Zuhra Hashimi - Portfolio</title>
+</head>
+
+<body>
+    <div>
+    <section>
+        <header>
+            <div>
+            <nav class="topnav">
+            
+                <a href="#home" class="logo">Zuhra Hashimi</a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+        </header>
+    </section></div>
+
+    <section class="main-content">
+        <div class="container">
+            
+            <h1 class="container headline">Hey There.<br>I'm Zuhra</h1>
+            <h2 class="container job-title">I am a web full stack developer </h2>
+            <p class="container job-description">I can help you build a product,<br>
+                feature or website.Look through some<br>
+                of my work and experience! If you like<br>
+                 what you see and have a project you need coded,<br>
+                  do not hestiate to contact me.</p>
+        </div>
+    
+        <div class="links">
+            <ul class="social_media">
+                <li>
+                    <a href="https://github.com/ZuhRa-HashiMi" id="git" target="blank"><img src="assets/github.svg" alt="github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/zuhra-hashimi-601966214/" id="linkedin" target="blank"><img src="assets/linkedin.svg" alt="linkedin"></a>
+                </li>
```

Comment:
- [ ] Kindly this icon is not showing properly. check out the `src` and fix it as weel 
Your design
![Screenshot from 2022-05-31 23-20-40](https://user-images.githubusercontent.com/77842237/171277735-12ebfa26-e752-4a56-b48d-df6cad5499e9.png)

Reply:
Thanks so much.
I did all of them